# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: e3051b96067a
# Full Hash: e3051b96067a480b020bd8c984a54491665c765e
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2021-11-11 15:25:44
# Regressor Bug: 1739448
# File Overlap Count: 1
# Description:
#   Bug 1739448 - Implement a prototype WebGL-backed Canvas2D. r=gfx-reviewers,nical,jrmuizel
#   
#   This mainly provides DrawTargetWebgl, which implements the subset of the DrawTarget
#   API necessary for integration with CanvasRenderingContext2D. It translates them to
#   suitable commands for its internal ClientWebGLContext, which then manages remoting
# ==============================================================================

diff -r 66c30950518d -r e3051b96067a modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Thu Nov 11 07:16:57 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Thu Nov 11 07:16:58 2021 +0000
@@ -4839,6 +4839,41 @@
   value: 0
   mirror: once
 
+- name: gfx.canvas.accelerated
+  type: RelaxedAtomicBool
+  value: false
+  mirror: always
+
+- name: gfx.canvas.accelerated.cache-items
+  type: RelaxedAtomicUint32
+  value: 1024
+  mirror: always
+
+- name: gfx.canvas.accelerated.cache-size
+  type: RelaxedAtomicUint32
+  value: 128
+  mirror: always
+
+- name: gfx.canvas.accelerated.max-size
+  type: RelaxedAtomicInt32
+  value: 0
+  mirror: always
+
+- name: gfx.canvas.accelerated.min-size
+  type: RelaxedAtomicInt32
+  value: 128
+  mirror: always
+
+- name: gfx.canvas.accelerated.max-surface-size
+  type: RelaxedAtomicUint32
+  value: 5280
+  mirror: always
+
+- name: gfx.canvas.accelerated.shared-page-size
+  type: RelaxedAtomicUint32
+  value: 1024
+  mirror: always
+
 # 0x7fff is the maximum supported xlib surface size and is more than enough for canvases.
 - name: gfx.canvas.max-size
   type: RelaxedAtomicInt32
